e2e/app/eoa/solver.go

```diff
@@ -0,0 +1,81 @@
+package eoa
+
+import (
+	"math/big"
+
+	"github.com/omni-network/omni/lib/evmchain"
+	"github.com/omni-network/omni/lib/netconf"
+	"github.com/omni-network/omni/lib/tokens"
+
+	"github.com/ethereum/go-ethereum/params"
+)
+
+type SolverNetThreshold struct {
+	minGwei uint64
+}
+
+func (t SolverNetThreshold) MinBalance() *big.Int {
+	return new(big.Int).Mul(big.NewInt(params.GWei), new(big.Int).SetUint64(t.minGwei))
+}
+
+var (
+	// solverThresholds defines the solvernet  thresholds RoleSolver: network -> chain -> token -> threshold.
+	solverThresholds = map[netconf.ID]map[uint64]map[tokens.Token]SolverNetThreshold{
+		netconf.Mainnet: {
+			evmchain.IDEthereum: {
+				tokens.WSTETH: {minGwei: gwei(10)}, // 10 wstETH
```

Comment:
This confuses me: the comment says `10 wstETH`, but the function name suggests we'll get `10_000_000_000`?

Reply:
Same below.